change docusaurus admonitions to our style of admonitions (#14917)
netdata/netdata: docs/cloud/manage/plans.md

We also provide paid subscriptions that designed to provide additional features and capabilities for businesses that need tighter and customizable integration of the free monitoring solution to their processes. These are divided into three different plans: **Pro**, **Business**, and **Enterprise**. Each plan will offers a different set of features and capabilities to meet the needs of businesses of different sizes and with different monitoring requirements.

> ### Note

> > To not disrupt current users space's access rights we will be keeping their current plan as the **Early Bird** plan. The reason for this is to allow users to

Reviewer: ```suggestion
> To not disrupt the existing space user's access rights we will keep them in the **Early Bird** plan. The reason for this is to allow users to
```
